warehouse/query-core/src/main/java/datawave/query/attributes/Document.java

```diff
@@ -212,16 +212,16 @@ public Attribute<?> toDocKeyAttributes(Set<Key> docKeys, boolean keepRecordId) {
     public void debugDocumentSize(Key docKey) {
         long bytes = sizeInBytes();
         // if more than 100M, then error
-        if (bytes > (ONE_HUNDRED_M)) {
-            log.error("Document " + docKey + "; size = " + size() + "; bytes = " + bytes);
-        }
+        // if (bytes > (ONE_HUNDRED_M)) {
+        // log.error("Document " + docKey + "; size = " + size() + "; bytes = " + bytes);
+        // }
         // if more than 10M, then warn
         // else if (bytes > (1024l * 1000 * 10)) {
         // log.warn("Document " + docKey + "; size = " + size() + "; bytes = " + bytes);
         // }
```

Comment:
if the code needs to be removed, remove it. don't leave commented out code.

Reply:
updated